### ISSUE TITLE
Support for Ubuntu 24.04

### ISSUE THREAD

**Issue by dmmeteo** [dokku/dokku#7000]
### Description of feature

Currently, I receive the following message when I try to install Dokku on the new Ubuntu server:

```
Unsupported Linux distribution. Only the following versions are supported: Debian [10, 11, 12], Ubuntu [18.04, 20.04, 22.04]
```

Is there a plan to extend support to Ubuntu 24.04? This would help in maintaining up-to-date server environments and ensure compatibility with the latest Ubuntu version. Thank you for considering this feature request.

**Comment by HectorMalot** [comment]
There is an ongoing issue already on Ubuntu 24.04 support: https://github.com/dokku/dokku/issues/6856 

Almost there. Herokuish needs to be released for 24.04 and then dokku can support 24.04 as well.

**Comment by josegonzalez** [comment]
Closing as duplicate.
